portfolio/src/main/webapp/comments-script.js

```diff
@@ -16,28 +16,37 @@
  * Fetches the comments input and builds the UI into a list.
  */
 
+var row;
 function getComments() {
-    const responsePromise = fetch('/comments');
-    responsePromise.then(handleResponse);
+    const RESPONSE_PROMISE = fetch('/comments');
+    RESPONSE_PROMISE.then(handleResponse);
 }
 
 function handleResponse(response) {
-    const JsonPromise = response.json();
-    JsonPromise.then(addCommentToDom);
+    const JSON_PROMISE = response.json();
+    JSON_PROMISE.then(addCommentToDom);
 }
 
 function addCommentToDom(comments) {
-    const commentsElement = document.getElementById('comments-container');
-    commentsElement.innerHTML = " ";
+    const TABLE_ELEMENT = document.getElementById('comments-table');
+    TABLE_ELEMENT.innerHTML = '<tr><th>Name</th><th>Message</th></tr>';
```

Comment:
Here too. You declare it as a constant and then immediately re-assign a sub-object. It is not deeply immutable.